src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -135,18 +136,26 @@ StackValue Pop()
 
         public ILImporter(MethodDesc method, MethodIL methodIL)
         {
-            _method = method;
             _typeSystemContext = method.Context;
 
+            // Instantiate method and its owning type
+            var instantiatedType = method.OwningType;
+            _method = method;
```

Comment:
Nit: It may look better if there is also `instantiatedMethod` local - so that it is symmetric with instantiatedType